### DISCUSSION TITLE
How to get the index of upscaler added later (4x UltraSharp)

### DISCUSSION THREAD

**Discussion by menguzat** [Coyote-A/ultimate-upscale-for-automatic1111#135] [Q&A]
I added an upscaling script called 4xUltraSharp.
I'm using ultimate upscale through the webui api
the readme shows the default scripts, but how do i access the added script? is the index the same as the order in which it is displayed on the web ui? or else, how do i access it?

**Comment by JAUKYU** [comment]
https://civitai.com/models/116225/4x-ultrasharp dowload this file and follow intruction 

HOW TO INSTALL:
Rename the file from:

4x-UltraSharp.pt to: 4x-UltraSharp.pth

Copy the file 4x-UltraSharp.pth inside creat the new folder ESRGAN : "\YOUR~STABLE~DIFFUSION~FOLDER\models\ESRGAN\")

Restart you Stable Diffusion

Now in "Extra Tab" you got the new upscaler
